lib/zendesk_apps_support/assets/installed.js.erb

```diff
@@ -14,4 +14,8 @@
   <% end %>
 }());
 
+<% unless rollbar_zaf_access_token.empty? %>
```

Comment:
@sandlerr it defaults to an `empty` string [here](https://github.com/zendesk/zendesk_apps_support/pull/157/files#diff-67a579144c0a408fe3168e4a92bf4420R19).